contracts/market/template/MarketCore.sol

```diff
@@ -629,7 +627,7 @@ contract MarketCore is AccessControlledAndUpgradeableModifiers, IMarketCommon, I
   }
 
   /// @notice Place the market in a state where no more price updates or mints are allowed
-  function deprecateMarket() external adminOnly {
+  function deprecateMarket() external emergencyMitigationOnly {
```

Comment:
This new role can deprecate markets quicker.

`Admin` will be under a > 7 day timelock, so won't be able to respond quickly enough if something like this is needed.